setup.py

```diff
@@ -1508,7 +1508,8 @@ def build_extensions(self):
 tensorflow_gpu_require_list = ['tensorflow-gpu']
 keras_require_list = ['keras>=2.0.8,!=2.0.9,!=2.1.0,!=2.1.1']
 pytorch_require_list = ['torch']
-mxnet_require_list = ['mxnet>=1.4.1']
+mxnet_require_list = ['mxnet-cu102>=1.4.1']
```

Comment:
I wonder if this might confuse users, since the extra doesn't align with the actual package.

Maybe we can do this instead:

```
mxnet: mxnet>=1.4.1
mxnet-cu102: mxnet-cu102>=1.4.1
```

What do you think?

Reply:
That makes sense. Just updated the code.